### ISSUE TITLE
Example of "aria-disabled" for Toolbar not for buttons only

### ISSUE THREAD

**Issue by abaksha-sc** [w3c/aria-practices#1040]
Provided example of Toolbar says that disabled button should be focusable (to make it possible use  attribute 'aria-disabled' instead of 'disabled'). But what about other controls (selects, checkboxes etc.)? Should they be focusable too?
For example, when I disable checkbox 'Night Mode' I can't focus on it.
